tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_consolidation_request.py

```diff
@@ -797,6 +942,156 @@ def test_incorrect_unknown_target_pubkey(spec, state):
     )
 
 
+@with_electra_and_later
+@spec_state_test
+def test_switch_to_compounding_exited_source(spec, state):
+    # Set up an otherwise correct request
+    current_epoch = spec.get_current_epoch(state)
+    source_index = spec.get_active_validator_indices(state, current_epoch)[0]
+    source_address = b"\x22" * 20
+    set_eth1_withdrawal_credential_with_balance(
+        spec, state, source_index, address=source_address
+    )
+    consolidation = spec.ConsolidationRequest(
+        source_address=source_address,
+        source_pubkey=state.validators[source_index].pubkey,
+        target_pubkey=state.validators[source_index].pubkey,
+    )
+
+    # exit source
+    spec.initiate_validator_exit(state, source_index)
+
+    # Check the the return condition
```

Comment:
This typo is present in all of the failing tests.
```suggestion
    # Check the return condition
```